how-to/how-to-set-prometheus-and-grafana.md

```diff
@@ -0,0 +1,82 @@
+# How to setup Prometheus and `Grafana` as observability platform
+
+Prometheus and `Grafana` are open sources analytics & monitoring solution. This how-to will introduce a simple setup for one Linux node configuration, all Kata Containers, Prometheus and `Grafana` are running on the same host.
+
+> **Warning**: This how-to is only for evaluation purpose, you **SHOULD NOT** running it in production.
+
+* [Install Prometheus](#install-prometheus)
+* [Install `Grafana`](#install-grafana)
+* [Setup `Grafana`](#setup-grafana)
+  * [Create `datasource`](#create-datasource)
+  * [Import dashboard](#import-dashboard)
+
+Both Prometheus and `Grafana` can installed using pre-compiled distributions.
+
+## Install Prometheus
+
+Download and extract Prometheus.
+
+```
+$ export VERSION=2.18.1
+$ curl -sL https://github.com/prometheus/prometheus/releases/download/v${VERSION}/prometheus-${VERSION}.linux-amd64.tar.gz -o prometheus-${VERSION}.linux-amd64.tar.gz
+$ tar xvfz prometheus-${VERSION}.linux-amd64.tar.gz
+$ cd prometheus-${VERSION}.linux-amd64
+```
+
+To let Prometheus to collect metrics from Kata Containers, first add `magent` to a Prometheus target.
```

Comment:
To enable Promethus collection of Kata Container metrics, add `magent` as a `Prometheus` target.